Makefile

```diff
@@ -33,7 +33,7 @@ bazel-push-images:
 push: bazel-push-images
 
 bazel-test:
-	hack/dockerized "hack/bazel-fmt.sh && CI=${CI} ARTIFACTS=${ARTIFACTS} hack/bazel-test.sh"
+	hack/dockerized "hack/bazel-fmt.sh && CI=${CI} ARTIFACTS=${ARTIFACTS} GO_TEST_TARGET=${GO_TEST_TARGET}  hack/bazel-test.sh"
```

Comment:
target `go-test:` has a variable called `${WHAT} for this purpose, maybe we can be consistent here?

Reply:
yes, you are right let me change it

Reply:
and the description of the PR